test/TestUtils/TestProjectFixture.cs

```diff
@@ -0,0 +1,290 @@
+﻿using Microsoft.DotNet.Cli.Build;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.DotNet.CoreSetup.Test
+{
+    public class TestProjectFixture
```

Comment:
This represents an abstraction around a test project which handles setting it up, mutating it to cover additional scenarios, and copying things to the right places whilst preventing duplication.

It represents a single test project and is not a framework, so fixture seemed fitting. Do you have an alternative suggestion?

---
In reply to: [66102620](https://github.com/dotnet/core-setup/pull/97#discussion_r66102620) [](ancestors = 66102620)